### PR TITLE
Feature/clients

### PR DESCRIPTION
**Pull request by mccleaveryan**
- Stubbed methods for clients
- Repo calls and service methods to reflect Swagger and those within Java back end of adminCore

### CODE REVIEW

File: v2/AdminCore.WebApp/Controllers/ClientController.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    [HttpGet]
+    [AllowAnonymous]
+    public ActionResult GetClientById(int clientId)
+    {
+      _clientService.GetClientById(clientId);
+      return Ok();
+    }
+
+    [HttpGet]
```
Comment (BrownMark) [inline v2/AdminCore.WebApp/Controllers/ClientController.cs]: We'll need a specific route here.

File: v2/AdminCore.WebApp/Controllers/ClientController.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    [HttpDelete]
+    [AllowAnonymous]
+    public ActionResult DeleteClient([FromBody] ClientViewModel clientViewModel)
+    {
+      _clientService.DeleteClient(clientViewModel.ClientId);
+
+      return Ok();
+    }
+
+    [HttpGet]
```
Comment (BrownMark) [inline v2/AdminCore.WebApp/Controllers/ClientController.cs]: We'll need a specific route here.

**Review by BrownMark — CHANGES_REQUESTED** [review]
Good work, just minor changes.

File: v2/AdminCore.Common/Interfaces/IClientService.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    void CreateClient(ClientDto clientDto);
+
+    void SaveClient(ClientDto clientDto);
+
+    void DeleteClient(int clientId);
+
+    ClientDto GetClientById(int clientId);
+
+    ClientDto GetClientByClientNameContaining(string clientName);
```
Comment (eoinmcafee) [inline v2/AdminCore.Common/Interfaces/IClientService.cs]: Not sure about the naming here, "containing". Should be fine just to say GetClientByClientName

**Review by BrownMark — APPROVED** [review]
Good work

**Comment by eoinmcafee** [comment]
Nice and tidy code @mccleaveryan. Good job! happy to merge

**Pull request merged**